azure/Guidelines.md

```diff
@@ -492,6 +492,12 @@ This indicates to client libraries and customers that values of the enumeration
 <a name="json-document-extensible-enums"></a>
 :white_check_mark: **DO** document to customers that new values may appear in the future so that customers write their code today expecting these new values tomorrow.
 
+<a name="json-return-extensible-enum-value"></a>
+:heavy_check_mark: **YOU MAY** return a value for an extensible enum that is not one of the values defined for the api-version specified in the request.
+
+<a name="json-accept-extensible-enum-value"></a>
+:warning: **YOU SHOULD NOT** accept a value for an extensible enum that is not one of the values defined for the api-version specified in the request.
+
 <a name="json-removing-enum-value-is-breaking"></a>
 :no_entry: **DO NOT** remove values from your enumeration list as this breaks customer code.
```

Comment:
Just so we're all clear: A service no longer supporting a value is a breaking change and this is allowed following the breaking change policy. In this case, all bets are off though and so the enum value CAN be removed during a formal breaking change.

Reply:
Yes, this guideline can be violated by declaring a formal breaking change. But I don't think we want to soften the guidance.

We have allowed some breaking change guidance to seep into this doc. In some ways it is unavoidable given that at least parts of our guidance are motivated by avoiding breaking changes later.